flocker/node/agents/ebs.py

```diff
@@ -29,6 +32,45 @@
 CLUSTER_ID_LABEL = u'flocker-cluster-id'
 ATTACHED_DEVICE_LABEL = u'attached-device-name'
 BOTO_NUM_RETRIES = u'10'
+BOTO_DEBUG_LEVEL = u'1'
+
+# Scaffolding for logging Boto client and server exceptions
+# via Eliot.
+CODE = Field.for_types("code", [int], u"The HTTP response code.")
+MESSAGE = Field.for_types(
+    "message", [bytes, unicode],
+    u"A human-readable error message given by the response.",
+)
+DETAILS = Field.for_types("details", [dict], u"Extra details about the error.")
+REQUEST_ID = Field.for_types(
+    "request_id", [bytes, unicode],
+    u"The unique identifier assigned by the server for this request.",
+)
+URL = Field.for_types("url", [bytes, unicode], u"The request URL.")
+METHOD = Field.for_types("method", [bytes, unicode], u"The request method.")
+
+RESPONSE = Field.for_types("response", [bytes, unicode], u"The response body.")
+BOTO_CLIENT_ERROR = MessageType(
+    u"boto:boto_client_error", [
+        CODE,
+        RESPONSE,
+        MESSAGE,
+        DETAILS,
+        REQUEST_ID,
+        URL,
+        METHOD,
+    ],
```

Comment:
:question: It'd be helpful to add a description, explaining when / where / how this error might be expected to occur.
 * http://eliot.readthedocs.org/en/0.7.1/types.html#message-types